staff/clara-rufi/cv/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Clara Rufí</title>
+</head>
+    <header>
+    <h1>Clara Rufí</h1>
+    <figure>
+    <img src="" alt="_Perfil picture">
+    </figure>
+   
+    <div id= "personal information:"></div>
+    <div id= "informationP">Age: 33</div>
+    <div id= "informationP">ID: 47732058C</div>
+    <div id= "informationP">mobile: 617555958</div>
+    <div id= "informationP">email: clara_rufi@hotmail.com</div>
+    <a href="www.clararufi.com">www.clararufi.com</a>
+    </header>
+
+    <br>
+
+    <h3>Education:</h3>
+    <ul>
+        <li>2017. Training certificate Training certificate BacGene Salmonella spp -PCR kit for the detection of Salmonella spp. </li>
+        <li>2017. GMP - Quality system of the pharmaceutical sector</li>
+        <li>2007-2010. Science and Food Technology. UB</li>
+        <li>2003-2007. Nutrition. UVic</li>
+    </ul>
+    <br>
+
+    <h3>Work experience:</h3>
+       
+    <h4><ul> 2010 - 2012 Bactereco. Analista d’aliments i d’aigües. Sant Feliu de Llobregat</h4></ol>
+        <ul>
+            <ul>  
+                <li> Microbiological analysis of food and water. Expression of results. Traceability of all the means and utensils used.</li>
+                <li> Quality assurance exercises (precision, repetitiveness and detection limit)</li>
+                <li> Preparation of sterile material and culture media. Stock control</li>
+            </ul>
+        </ul>
+    <h4><ul> 2012 - Bactereco.Quality assurance</h4></ul>
+        <ul>
```

Comment:
unordered list inside another it's not recommended

Reply:
Thanks for the suggestion. I have corrected it